common/src/main/java/tachyon/client/WorkerBlockMasterClient.java

```diff
@@ -148,19 +148,17 @@ public synchronized Command heartbeat(long workerId, List<Long> usedBytesOnTiers
    * @param usedBytesOnTiers list of the used byes on each tier
    * @param currentBlocksOnTiers a mapping of each storage dir, to all the blocks on that storage
    *        dir
-   * @return the worker id
    * @throws IOException if an I/O error occurs
    */
   // TODO: rename to workerBlockReport or workerInitialize?
-  // TODO(cc) return value should be void, throw exception when workerId doesn't exist.
-  public synchronized long register(long workerId, List<Long> totalBytesOnTiers,
+  public synchronized void register(long workerId, List<Long> totalBytesOnTiers,
```

Comment:
Yes, the "@throws" line should say it is thrown when the workerid does not exist.